file/delete_scheduler_test.cc

```diff
@@ -445,6 +445,11 @@ TEST_F(DeleteSchedulerTest, StartBGEmptyTrashMultipleTimes) {
   rate_bytes_per_sec_ = 1024 * 1024;  // 1 MB / sec
   NewDeleteScheduler();
 
+  // If trash file is generated faster than deleting, delete_scheduler will
+  // delete it directly instead of waiting for background trash empty thread to
+  // clean it. Set the ratio higher to avoid that.
+  sst_file_mgr_->SetMaxTrashDBRatio(10);
```

Comment:
Sounds like the chosen ratio `10` depends on the number of iterations `5`? If there are more iterations, we will be generating more trash. Suppose trash empty thread lags behind, then it will lag behind even more with more iterations. Therefore, we need to increase the trash/db ratio even more?

Reply:
Good point, we should use a const variable for that.
It should only depend on the dummy files number inside of each run, which is 10: https://github.com/facebook/rocksdb/blob/5cdc3902b6ab81d883b974cb3263b9944f264208/file/delete_scheduler_test.cc#L455-L456
Each run should be indecent, because it `delete_scheduler_->WaitForEmptyTrash();` (and the test assumes all deleted file should go to trash).